app/components/submission/FileUploads/FileUploads.js

```diff
@@ -6,10 +8,23 @@ import { FormH2 } from '../../ui/atoms/FormHeadings'
 import ButtonLink from '../../ui/atoms/ButtonLink'
 import FileUpload from '../../ui/molecules/FileUpload'
 import ValidatedField from '../../ui/atoms/ValidatedField'
+import IconTextBold from '../../ui/atoms/icons/TextBold'
```

Comment:
Add this to index.js to do a single import of multiple components?

Reply:
This will need to be done to support icons in themes but as it stands the `Icon` component applies styles which mess up these icons.